### PR TITLE
Feature/moz 413 wrap event strings

### PR DESCRIPTION
**Pull request by kycapstick**

### CODE REVIEW

File: plugins/events-manager/forms/event-editor.php [hunk 1]
```diff
@@ -92,7 +92,7 @@
 				<div class="event-creator__three-up">
 					<div class="half">
 						<label class="event-form-details event-creator__label" for="event-description"><?php print __('Event description *', 'commuity-portal'); ?></label>
-						<textarea name="content" id="event-description" placeholder="Add in the details of your event’s agenda here. If this is a multi-day event, you can add in the details of each day’s schedule and start/end time." rows="10" id="event-description" class="event-creator__input event-creator__textarea" style="width:100%" required maxlength="3000"><?php echo __($EM_Event->post_content) ?></textarea>
+						<textarea name="content" id="event-description" placeholder="<?php  print __('Add in the details of your event’s agenda here. If this is a multi-day event, you can add in the details of each day’s schedule and start/end time.', 'community-portal') ?>" rows="10" id="event-description" class="event-creator__input event-creator__textarea" style="width:100%" required maxlength="3000"><?php echo __($EM_Event->post_content) ?></textarea>
```
Comment (dnaluz) [inline plugins/events-manager/forms/event-editor.php:95]: The content here doesn't need to be wrapped in gettext

File: plugins/events-manager/forms/event/location-moz.php [hunk 2]
```diff
@@ -9,68 +9,69 @@
...
-      </div>
-    </div>
-    <div class="event-creator__three-up">
-      <div class="wide">
-        <label id="location-country-label" class="event-creator__label" for="location-country"><?php _e ( 'Where will this event be held? *', 'commuity-portal')?></label>
-        <select class="event-creator__dropdown" id="location-country" name="location_country" <?php if ($event) : echo esc_attr("disabled"); endif; ?> required>
+		?>
+		<div class="event-creator__three-up">
+			<div class="wide <?php echo ($event) ? esc_attr("wide--md-third") : null ?>">
+				<label class="event-creator__label" for="location-type"><?php echo __('Is this event online or on location? *') ?></label>
```
Comment (dnaluz) [inline plugins/events-manager/forms/event/location-moz.php:33]: This is missing the community-portal namespace

File: plugins/events-manager/forms/event-editor.php [hunk 3]
```diff
@@ -92,7 +92,7 @@
 				<div class="event-creator__three-up">
 					<div class="half">
 						<label class="event-form-details event-creator__label" for="event-description"><?php print __('Event description *', 'commuity-portal'); ?></label>
-						<textarea name="content" id="event-description" placeholder="<?php  print __('Add in the details of your event’s agenda here. If this is a multi-day event, you can add in the details of each day’s schedule and start/end time.', 'community-portal') ?>" rows="10" id="event-description" class="event-creator__input event-creator__textarea" style="width:100%" required maxlength="3000"><?php echo __($EM_Event->post_content) ?></textarea>
+						<textarea name="content" id="event-description" placeholder="Add in the details of your event’s agenda here. If this is a multi-day event, you can add in the details of each day’s schedule and start/end time." rows="10" id="event-description" class="event-creator__input event-creator__textarea" style="width:100%" required maxlength="3000"><?php echo __($EM_Event->post_content) ?></textarea>
```
Comment (dnaluz) [inline plugins/events-manager/forms/event-editor.php]: Placeholder needs to be wrapped here

Comment (dnaluz) [reply plugins/events-manager/forms/event-editor.php]: Also post_content doesn't need to be wrapped

Comment (kycapstick) [reply plugins/events-manager/forms/event-editor.php]: Sorry! Misread the first comment thought it was referring to the placeholder. This has been switched back

**Comment by dnaluz** [comment]
Looks good

**Pull request merged**